ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__metal-4.16-nightly-x86.yaml

```diff
@@ -31,8 +31,7 @@ tests:
       BAREMETAL: "true"
       KUBE_BURNER_VERSION: 1.2.9
       NUM_WORKER_NODES: "6"
-      OCP_RELEASE_IMAGE: quay.io/openshift-release-dev/ocp-release:4.16.0-rc.9-x86_64
-      OCP_VERSION_SHORT: "4.16"
+      OCP_VERSION: latest-4.16
```

Comment:
This looks like a development branch job based on its name, shouldn't you set OCP_BUILD to "dev" here?

Reply:
I was under the impression that `latest-` is reserved for ga builds and `candidate-` for dev builds, is it not the case?

Reply:
dev supports all tags here: https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp-dev-preview/ Including things like candidate-4.16, candidate-4.17 which will have the latest/last ec release and if you just use "latest" it points to the ec of the currently in dev release.

For ga you can have things like fast-4.16, stable, 4.16 or latest-4.16 or even candidate-4.16 (which I believe has the candidate z stream). Any directory on this link is fair game- https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/

Reply:
So, we should rename this from `nightly` to `candidate`?  for 4.16

Reply:
@smalleni Can you share with us how to capture a nightly build of a OCP version? For example, 4.16 nightly using the links above?